### PR TITLE
Pridat sumu hodin do week view

### PR DESCRIPTION
**Pull request by MichalMartinek**

### CODE REVIEW

File: src/app/week-log/week-log.component.spec.ts [hunk 1]
```diff
@@ -22,4 +70,28 @@ describe('WeekLogComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should calculate totalSum', () => {
+    expect(component.totalSum).toBe(12.25);
```
Comment (dborosova) [inline src/app/week-log/week-log.component.spec.ts]: Some comment why is it so would be helpful. Or change of description.

**Comment by dborosova** [comment]
Works fine, but I would recommend rework style of the summary row in order to improve UX.
image

**Comment by MichalMartinek** [comment]
@dborosova I rewrited the tests and added more styles to the summary row. 
Screenshot from 2019-07-30 20 38 30
Is it better?

File: src/app/week-log/week-log.component.spec.ts [hunk 2]
```diff
@@ -46,14 +46,12 @@ describe('WeekLogComponent', () => {
     expect(component).toBeTruthy();
   });
 
-  it('should calculate totalSum', () => {
-    expect(component.totalSum).toBe(dayLog1.hoursLogged + dayLog2.hoursLogged);
-  });
-
-  it('should display totalSum', () => {
+  it('should calculate correctly total sum of logged hours and display it', () => {
+    const totalSum = dayLogs.reduce((sum, val) => sum + val.hoursLogged, 0);
```
Comment (dborosova) [inline src/app/week-log/week-log.component.spec.ts]: It is handy to prefix expected values, e.g. with 'expected' in order to distinguish values obtained and expected values.

File: src/app/week-log/week-log.component.spec.ts [hunk 3]
```diff
@@ -20,7 +20,7 @@ class MockRedmineService extends RedmineService {
   }
 }
 
-describe('WeekLogComponent', () => {
+fdescribe('WeekLogComponent', () => {
```
Comment (dborosova) [inline src/app/week-log/week-log.component.spec.ts]: Why 'fdescribe'?

File: src/app/week-log/week-log.component.ts [hunk 4]
```diff
@@ -54,6 +54,7 @@ export class WeekLogComponent implements OnInit {
   listWorkingDayLogsForWeek() {
     this.redmine.listDayLogs(this.weekHtml5fmt, 'week', true, true, false).subscribe(dayLogs => {
       this.dayLogs = dayLogs;
+      console.log(dayLogs)
```
Comment (dborosova) [inline src/app/week-log/week-log.component.ts]: Do not log whole objects

**Pull request merged**